content/handbook/support/support-engineer-responsibilities.md

```
@@ -0,0 +1,369 @@
+
+オンボーディング中に、最初の注力領域が明確に示されます。
+
+このページを読む際は、現在の注力領域を念頭に置いてください。これにより、正しいワークフローとロールを見つけるのに役立ちます。
+
+## サポートエンジニアのコアな責務
+
+GitLab サポートエンジニアのコアな責務は、サポートチケットで提示される問題を解決し、質問に答えるために顧客と協力することで、日々顧客に優れたサポート体験を提供することです。
+
+この責務は 5 つの主要なコンポーネントに分けることができます:
+
+1. [適切に対応可能であること](#1-be-appropriately-available)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)